stencil.conf.js

```diff
@@ -27,57 +17,60 @@ var watchOptions = {
         '/assets/scss',
         '/assets/less',
         '/assets/css',
+        '/assets/dist',
     ]
 };
 
 /**
- * Hook into the stencil-cli browsersync instance for rapid development of themes.
  * Watch any custom files and trigger a rebuild
- * @param {Object} Bs
  */
-function development(Bs) {
-    var compiler = webpack(webpackConfig);
-
+function development() {
     // Rebuild the bundle once at bootup
-    compiler.watch({}, err => {
+    webpack(webpackConfig).watch({}, err => {
         if (err) {
             console.error(err.message, err.details);
         }
 
-        Bs.reload();
+        process.send('reload');
     });
 }
 
 /**
-
-*/
-/**
  * Hook into the `stencil bundle` command and build your files before they are packaged as a .zip
- * Be sure to call the `done()` callback when finished
- * @param {function} done
  */
-function production(done) {
-    var compiler;
-
+function production() {
+    webpackConfig.watch = false;
     webpackConfig.devtool = false;
-    webpackConfig.plugins.push(new webpack.optimize.DedupePlugin());
     webpackConfig.plugins.push(new webpack.optimize.UglifyJsPlugin({
-        comments: false
+        comments: false,
+        compress: {
+            warnings: true,
+        },
+        sourceMap: true,
```

Comment:
If this plugin is only used to bundle for production we don't include the source maps